man/logind.conf.xml

```diff
@@ -355,6 +355,19 @@
         <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>.
         </para></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>MaintenanceWindow=</varname></term>
```

Comment:
I think this should convey in some way that this expects a time/calendar value. "window" sounds like a misnomer to me, since it suggests a start+stop value, but actually this instead expects a possibly repetitive calendar time expression, i.e. a series of points in times, not a timespan

Reply:
Any suggestions? PreferredShutdownTime?

Reply:
Maybe `DesignatedMaintenanceTime=`?